tsconfig.base.json

```diff
@@ -6,7 +6,7 @@
     "declaration": true,
     "declarationMap": true,
     "esModuleInterop": true,
-    "lib": ["ES2021"],
+    "lib": ["ES2023"],
```

Comment:
I was going to ask why ES2023 was legal, but node.green does in fact show that all versions of Node 18 support all runtime features of ES2023. I've fixed https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping to reflect this (which matches the bases too).

Reply:
Nice!!